lib/vanagon/engine/pooler.rb

```diff
@@ -35,12 +48,12 @@ def select_target
         response = Vanagon::Utilities.http_request(
           "#{@pooler}/vm",
           'POST',
-          '{"' + @platform.vmpooler_template + '":"1"}',
+          '{"' + @build_host_type + '":"1"}',
```

Comment:
This method shouldn't be called before the instance variable has been set, but just to be sure, I modified it to call the getter method instead.